.gitignore

```diff
@@ -4,3 +4,4 @@ dist
 build
 .vscode
 __pycache__
+cache
```

Comment:
Does it work when `cache` is a subdirectory? I'm more familiar with e.g. `**/cache/`

Reply:
It works, I tried doing `/tests/packages/**/cache` to catch only cache folders in packages but for some reason it didn’t work